src/ocamlorg_package/lib/ocamlorg_package.ml

```diff
@@ -15,11 +15,11 @@ type state = {
   version : string;
   mutable opam_repository_commit : string option;
   mutable packages : Info.t Version.Map.t Name.Map.t;
-  mutable stats : Packages_stats.t option;
-  mutable featured_packages : t list option;
+  mutable stats : Stats.t option;
+  mutable featured : t list option;
 }
 
-let state_of_package_list (pkgs : t list) =
+let state_of_list (pkgs : t list) =
```

Comment:
Here I'm not sold on shortening the name, however, I suppose that's because the old name is already quite nondescript. Maybe a better name is `opam_repository_state`?